### ISSUE TITLE
DAMSM validation loss increasing

### ISSUE THREAD

**Issue by dheerajiiitv** [chenqi008/BoGAN#6]
Hi Team,
Please help me here, I am training the image and text encoder, and the model word loss and sent loss is increasing after 3 epochs.
Let me know what could be the possible reason for this issue.
